addons/account/models/account_move_line.py

```diff
@@ -332,11 +332,11 @@ class AccountMoveLine(models.Model):
     # section related fields
     collapse_composition = fields.Boolean(
         string="Hide Composition",
-        help="If checked, the lines below this section will not be displayed in reports and portal.",
+        help="In the invoice, the customer see the section like a invoice line, all its content is hidden.",
```

Comment:
```suggestion
        help="In the invoice, the customer can see the invoice-line section, but all of its contents are hidden.",
```

Reply:
You can challenge it if don't feel it right instead just copy pasting it :see_no_evil: